internal/cmd/connect/command_plugin_install.go

```diff
@@ -0,0 +1,456 @@
+package connect
+
+import (
+	"archive/zip"
+	"bytes"
+	"crypto/md5"
+	"crypto/sha1"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type manifest struct {
+	Owner struct {
```

Comment:
Let's split this into individual structs? They might be more convenient to work with in the future if they're split up, and these nested structs have a tendency to get a bit out of hand if the schema ever evolves.

Reply:
What about one struct but flattened instead?

Splitting into multiple structs probably means passing at least 2 structs around in most places.

Reply:
You can still nest them, but when you define them you can define them separately.

Reply:
Ok. Does splitting out License and Archive sounds good?

Reply:
I was thinking owner, license, and archive should be defined separately since they're all structs